src/Assert.php

```diff
@@ -897,6 +897,11 @@ protected static function strlen($value)
         return mb_strwidth($value, $encoding);
     }
 
+    protected static function createException($message)
```

Comment:
Have you considered a name like `createInvalidArgumentException`? We may (though unlikely) want to introduce ie LogicExceptions in the future. That would make a name like `createException` be too vague.